src/lighting.h

```diff
@@ -57,4 +57,7 @@ struct Lighting
 	float saturation {1.0f};
 	float volumetric_light_strength {0.0f};
 	video::SColor shadow_tint {255, 0, 0, 0};
+	float bloom_intensity {0.05f};
```

Comment:
I opted for a default value of 0.05 (the old setting default) instead of 0 (would be consistent with other effects) because bloom intensity affects volumetric lighting strength, so changing the default to 0 now would break existing games/mods that enable volumetric lighting.

Reply:
Since volumetric lighting is fairly new (5.9) and missing it isn't game-breaking, we could also opt for consistency instead of compatibility here. Thoughts?

Reply:
I have a weak preference for leaving it at 0.05. It's a nice default effect.